apps/docs/src/web/AGENTS.md

```
@@ -145,6 +175,97 @@ function MyComponent() {
 }
 ```
 
+## SDK Explorer Components
```

markdownlint-cli2 (0.18.1)

Hard tabs
Column: 1

(MD010, no-hard-tabs)